Plugin~/WebRTCPlugin/GraphicsDevice/ITexture2D.h

```diff
@@ -0,0 +1,24 @@
+﻿#pragma once
+
+namespace WebRTC {
+
+struct ITexture2D {
+public:
+    //[TODO-Sin: 2019-19-11] ITexture2D should not be created directly, but should be called using
+    //GraphicsDevice->CreateEncoderInputTexture
+
+    ITexture2D(uint32_t w, uint32_t h) : m_width (w), m_height(h) {}
+    bool IsSize(uint32_t w, uint32_t h) { return m_width == w && m_height == h; }
```

Comment:
It seems better this method is declared as `const`

Reply:
ok